pos_partner_location_map/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "Point of Sale - Customer Screen",
+    "version": "16.0.1.0.0",
+    "category": "Point Of Sale",
+    "summary": "Point of Sale - Customers can easily interact with point of sale",
```

Comment:
Show and set partner location directly on map in POS

Reply:
Resolved